Earthfile

```diff
@@ -0,0 +1,290 @@
+VERSION --use-function-keyword 0.7
+FROM debian:bullseye-slim
+WORKDIR /app
+
+ARG --global RUST_VERSION=1.73.0
+ARG --global DISTROLESS_TAG=debug-nonroot
+ARG --global TARGETARCH
+ARG --global CARGO_PROFILE=dev
+ENV DENO_DIR /deno-dir/
+
+COPY_SOURCES_RUST:
```

Comment:
Earthly maintains a "lib" of helper function for rust: https://github.com/earthly/lib/tree/main/rust